apps/atomy-q/API/app/Services/EvidenceVault/SupportingEvidenceStorageService.php

```
@@ -0,0 +1,88 @@
+        array $relations = [],
+    ): SupportingEvidence {
+        $originalName = $this->sanitizeOriginalName($file->getClientOriginalName());
+        $path = sprintf(
+            'supporting-evidence/%s/%s/%s-%s',
+            $tenantId,
+            (string) $rfq->id,
+            (string) Str::ulid(),
+            $originalName,
+        );
+
+        $disk = Storage::disk('local');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Storage' in method 'store'. (undefined)

(StaticAccess)